proto/cometbft/types/v1/canonical.proto

```diff
@@ -40,7 +40,7 @@ message CanonicalVote {
   sfixed64                  height    = 2;  // canonicalization requires fixed size encoding here
   sfixed64                  round     = 3;  // canonicalization requires fixed size encoding here
   CanonicalBlockID          block_id  = 4 [(gogoproto.customname) = "BlockID"];
-  google.protobuf.Timestamp timestamp = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+//  google.protobuf.Timestamp timestamp = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
```

Comment:
This change should stay. Otherwise it's ugly to have two equivalent structs (the proto and the domain type) that don't have the same fields

Reply:
Also, can you reserve the field?